The Chapel Redivisioning

san7890 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Hello there, let's give you the best illustration of a problem. On IceBoxStation:

![image](https://user-images.githubusercontent.com/34697715/154592210-1cac5580-aae4-42b0-9942-9e34b49a8970.png)


That Chapel is massive. A lot of it is on the lower Z-level (that empty space), but it's still weird in that little part that pops up so you can send people out through the glorious mass driver is still just one more chunk of the already large Chapel.

This PR helps alleviate some of the stress on the APC caused during these services by adding a new area turf (Chapel Funeral Room), pathing, and changes to all of the four applicable maps (I already did Kilo back in #64847, similar but different).

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

![image](https://user-images.githubusercontent.com/34697715/154592249-f62723a7-e25c-4631-b46e-068e68b18c7e.png)

Noncontiguous areas should have their own APC, and it doesn't exactly make sense for two different sections of the same room on different Z-levels to share the same APC. This also helps fix blobbiness of the chapel on Delta, Meta, and Tram.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
add: On Meta, Delta, and Tram Stations, expect to see the word "Funeral" dance across your eyes a few times as you enter the chapel.
add: On IceBoxStation, the Chapel has undergone some new area definitions, and the wiring for the power to these areas have changed a bit from what you were accustomed to previously.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

san7890 committed:
The Chapel Redivisioning

Hello there, let's give you the best illustration of a problem. IceBoxStation:

That Chapel is massive. A lot of it is on the lower Z-level (that empty space), but it's still weird in that little part that pops up so you can send people out through the glorious mass driver is still just one more chunk of the already large Chapel.

This PR helps alleviate some of the stress on the APC caused during these services by adding a new area turf (Chapel Funeral Room), pathing, and changes to all of the four applicable maps (I already did Kilo back in ).

Maurukas reviewed (changes requested):
On Icebox I would strongly prefer that the entire multi-z chapel remain one area as it currently is. The area is quite interconnected and would prefer the whole thing to be handled by the same APC and fire alarms.

san7890 committed:
Undo IceBoxStation

Couldn't make it work.

san7890 committed:
The Chapel Redivisioning

Hello there, let's give you the best illustration of a problem. IceBoxStation:

That Chapel is massive. A lot of it is on the lower Z-level (that empty space), but it's still weird in that little part that pops up so you can send people out through the glorious mass driver is still just one more chunk of the already large Chapel.

This PR helps alleviate some of the stress on the APC caused during these services by adding a new area turf (Chapel Funeral Room), pathing, and changes to all of the four applicable maps (I already did Kilo back in ).

san7890 commented:
*shoots myself in the foot and adds fifteen people to this PR for review*

i don't know why github desktop does this to me

LemonInTheDark commented:
There's a simple solution to this problem
